foundry.toml

```diff
@@ -2,11 +2,13 @@
 src = 'contracts'
 out = 'foundry-out'
 solc_version = '0.8.20'
-optimizer_runs = 800
+via_ir = true
```

Comment:
oof is it super slow now

Reply:
kinda but not super? sub minute compile time 
had to do this otherwise stack too deep